assets/wizards/popups/views/popup-group/index.js

```diff
@@ -104,29 +126,98 @@ const PopupGroup = ( {
 	return (
 		<Fragment>
 			<div className="newspack-campaigns__popup-group__filter-group-wrapper">
-				<SelectControl
-					options={
-						-1 === campaignGroups
-							? [
-									{
-										value: -1,
-										label: __( 'Loading...', 'newspack' ),
-									},
-							  ]
-							: [
-									{ value: -1, label: __( 'All Campaigns', 'newspack' ) },
-									...campaignGroups.map( term => ( {
-										value: term.id,
-										label: term.name,
-									} ) ),
-							  ]
-					}
-					value={ campaignGroup }
-					onChange={ value => setCampaignGroup( +value ) }
-					label={ __( 'Groups', 'newspack' ) }
-					labelPosition="side"
-					disabled={ -1 === campaignGroups }
-				/>
+				<div className="newspack-campaigns__popup-group__filter-group-actions">
+					<SelectControl
+						options={
+							-1 === campaignGroups
+								? [
+										{
+											value: -1,
+											label: __( 'Loading...', 'newspack' ),
+										},
+								  ]
+								: [
+										{ value: -1, label: __( 'All Campaigns', 'newspack' ) },
+										...campaignGroups.map( term => ( {
+											value: term.id,
+											label: term.name,
+										} ) ),
+								  ]
+						}
+						value={ campaignGroup }
+						onChange={ value => setCampaignGroup( +value ) }
+						label={ __( 'Groups', 'newspack' ) }
+						labelPosition="side"
+						disabled={ -1 === campaignGroups }
+					/>
+					{ campaignGroup > 0 && (
+						<SegmentationPreview
+							campaignGroups={ campaignGroup }
+							segment={ segmentId }
+							renderButton={ ( { showPreview } ) => (
+								<div className="newspack-campaigns__popup-group__filter-group-segmentation">
+									{ activeCampaignGroup !== campaignGroup && (
```

Comment:
```suggestion
									{ +activeCampaignGroup !== campaignGroup && (
```

Reply:
I accidentally put this in a merge commit, but it's done in https://github.com/Automattic/newspack-plugin/pull/775/commits/99dd0a6a789c879b62a0e5a33643d1cda05aa0bb